release/models/sampling/openconfig-sampling-sflow.yang

```diff
@@ -146,6 +148,13 @@ module openconfig-sampling-sflow {
         sFlow collector.  If uspecified, the collector destination
         is reachable in the default network instance.";
     }
+
+    leaf originated-qos-marking {
```

Comment:
Oops, apologies, I missed that this was also included here vs. my proposal in #656. I'll close that PR. Should this just be `dscp`?

Also, should this be configured per collector? It looks from the survey in #656 that this isn't the common support, and rather it is should be globally set for all collectors.

Reply:
The only reason for the name was consistency to the only other published leaf today that represents the marking of sourced control/management traffic which falls in `openconfig-telemetry` today.  Since it is likely that the `openconfig-telemetry` model will evolve and iterate possibly spread to other domains, I'm fine w/ using slightly better terminology that can be brought across to other domains that lack this functionality but do exist in shipping implementations today.

Since IPv6 TC carries the 6-bits as "DSCP" the same as in IPv4, I'm fine changing this to just `dscp` - will push in next commit

Reply:
Also as far as setting DSCP globally vs. collector - this is a common vendor neutral issue where I do see a split of implementation behaviors.  This can be moved up to a global configuration and for the implementations that support per-collector would then mean all collectors inherit the same global value.

Reply:
I think this makes sense -- it seems unlikely to me that operationally one would need to specify different DSCP values for sFlow traffic based on collector.

Reply:
This has been updated to use a leaf name dscp which SGTM.

Reply:
@dplore - Actually - per this comment thread, commit https://github.com/openconfig/public/pull/522/commits/397a38c1af507d57476e8f2510af1ac1afd494c5 brought this to common global marking rather (where collectors would inherit the global marking)

```
module: openconfig-sampling
  +--rw sampling
     +--rw oc-sflow:sflow
        +--rw oc-sflow:config
        |  +--rw oc-sflow:dscp?   oc-inet:dscp
        +--rw oc-sflow:state
           +--rw oc-sflow:dscp?   oc-inet:dscp
```

Reply:
DSCP for EOS is set at a global level: https://www.arista.com/en/um-eos/eos-user-security#:~:text=switch(config)%23-,sFlow%20qos%20dscp,-36

So this LGTM